README.md

```diff
@@ -2,7 +2,7 @@
 
 [ISO 18013-5](https://www.iso.org/standard/69084.html) defines mDL (mobile Driver Licenses): an ISO standard for digital driver licenses.
 
-This is a Node.js library to issue and verify mDL [CBOR encoded](https://cbor.io/) documents.
+This is a Node.js library to issue and verify mDL [CBOR encoded](https://cbor.io/) documents in accordance with ISO 18013-7 (draft's date: 2023-08-02)"
```

Comment:
please update the samples/code snippets in this file too.

Reply:
👍🏼 see 11bbc76